src/core_plugins/kibana/public/management/sections/settings/advanced_row.html

```diff
@@ -166,6 +168,7 @@
           ng-if="!conf.editing"
           ng-click="clear(conf)"
           ng-hide="isDefaultValue(conf)"
+          aria-label="Clear {{conf.ariaName}}"
```

Comment:
Dunno. The button is labelled 'clear' so that's what I used for reference for the aria-label.